src/main/java/foodwhere/model/ModelManager.java

```diff
@@ -27,12 +27,12 @@ public class ModelManager implements Model {
     private final FilteredList<Review> filteredReviews;
 
     /**
-     * Initializes a ModelManager with the given addressBook and userPrefs.
+     * Initializes a ModelManager with the given addressBook in FoodWhere and userPrefs.
      */
     public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs) {
         requireAllNonNull(addressBook, userPrefs);
 
-        logger.fine("Initializing with address book: " + addressBook + " and user prefs " + userPrefs);
+        logger.fine("Initializing with stall list: " + addressBook + " and user prefs " + userPrefs);
```

Comment:
Perhaps "Initializing with data: " may be more accurate?

Reply:
Resolved in latest commit